STEERING-COMMITTEE.md

```diff
@@ -7,51 +7,55 @@ aliases:
+community, a healthy ecosystem of vendors, and most of all satisfied
+customers running their workloads on Knative clusters.
 
 The governance of Knative is an open, living document, and will continue to
 evolve as the community and project change. We expect over time we will adapt
 the way we run this committee, based on feedback from the community.
 
+- [Mission](#mission)
 - [Charter](#charter)
 - [Delegated authority](#delegated-authority)
 - [Committee Meetings](#committee-meetings)
 - [Committee Mechanics](#committee-mechanics)
 - [Committee Members](#committee-members)
-  - [Allocation of seats](#allocation-of-seats)
+  - [Composition](#composition)
+  - [Nomination of representatives](#nomination-of-representatives)
 - [Decision process](#decision-process)
 - [Changes to the charter](#changes-to-the-charter)
 - [Getting in touch](#getting-in-touch)
 
+## Mission
+
+The Steering Committee's mission is to guide the project such that:
+
+1. Knative is a truly open effort which is inclusive and empowers contributors
+1. Knative core is clearly defined with a conformance suite
+1. Trademark usage is monitored and violations addressed
+
 ## Charter
 
-1. Define, evolve, and promote the vision, values, mission, and scope of the
-   project.
-1. Define and evolve project governance structures and policies, including
-   project roles and how collaborators become members, approvers, leads, and/or
-   administrators. This includes policy for the creation and administration of
-   [working groups](./working-groups/WORKING-GROUPS.md) and committees.
-1. Steward, control access, delegate access, and establishes processes
-   regarding, all Knative project resources and has the final say in the
-   disposition of those resources.
-1. Manage the Knative brand and decide which things can be called "Knative" and
-   how that mark can be used in relation to other efforts or vendors.
+1. Define, evolve, and promote the values of the project, and address any
+   [Code of Conduct](./CODE-OF-CONDUCT.md) issues that may arise, and
+   maintain confidentiality.
+1. Review and approve changes to the scope of Knative core.
+1. Administer the marks in a transparent way, and take responsibility for
+   ensuring follow up on trademark violations.
 1. Confirm/reject nominations to the KSC from organizations who are allocated
    seats.
-1. Confirm/reject nominations to the Technical Oversight Committee.
-1. Receive and handle reports about [code of conduct](./CODE-OF-CONDUCT.md)
-   violations and maintain confidentiality.
-1. Receive security reports; work with the appropriate technical leads to accept
-   or reject the report; maintain the private nature of such reports until
-   disclosed to the broader community.
 1. Act as the final escalation point and decider for any disputes, issues,
    clarifications, or escalations within the project scope.
+1. Delegate all other responsibilities, including matters of governance
```

Comment:
It's not clear that the charter needs to indicate whether the Steering Committee will delegate or handle other items directly; this seems like an operating policy for the committee, rather than something to put in the charter.